### ISSUE TITLE
Camera in Resonite Essentials doesn't rotate portrait oriented pictures

### ISSUE THREAD

**Issue by Psychpsyo** [Yellow-Dog-Man/Resonite-Issues#205]
### Describe the bug?

When taking a picture with the default camera from the Resonite Essentials folder, the resulting picture won't actually be rotated by 90° when you rotated the camera while taking it.
This is annoying when UI-focusing onto the picture in desktop mode or when exporting it from resonite / when it shows up in your steam screenshots.

### To Reproduce

1. Get camera from Resonite Essentials
2. Turn it sideways
3. Take a picture of your friend (or anything, really)
4. Ctrl+click onto it.

### Expected behavior

The camera should create rotated pictures when it is rotated while taking the picture.

### Reporters

cameron5 on Discord, U-Psychpsyo in Resonite

**Comment by shadowpanther** [comment]
In-game this forces the Screen focus camera (Ctrl-click on said photo) to be sideways.

**Comment by AegisTheWolf** [comment]
I think adding not only a portrait mode would be good, I've been needing a square camera at times and had to make custom ones in the past, 

Basically, 3 base modes for aspect ratio and an option to do custom resolution for users to type into the camera menu.

**Comment by RyuviTheViali** [comment]
Okay, the camera has been updated to support other orientations! There is a new side menu called DIR which allows users to choose between automatic orientation and manual orientation choice. In automatic choice (default), holding the camera in a sideways orientation will take portrait photos. I've also pushed two new locale strings for this menu.

This should also fix the UI Focus issue mentioned in this thread.
